django_app/redbox_app/settings.py

```diff
@@ -19,21 +22,24 @@
 
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-COMPRESS_ENABLED = True
+COMPRESSION_ENABLED = env.bool("COMPRESSION_ENABLED")
+
 COMPRESS_PRECOMPILERS = (("text/x-scss", "django_libsass.SassCompiler"),)
 
 STATIC_URL = "static/"
-STATIC_ROOT = "frontend/"
+STATIC_ROOT = "staticfiles/"
```

Comment:
this appears to be the standard way of doing this: multiple directories to keep your static files but just one (different) `STATIC_ROOT` where they all get copied to on running `python manage.py collectstatic`

Reply:
Do we need to add `python manage.py collectstatic` to the `Makefile` somewhere?

Reply:
good idea